src/tools/wallmap/objloader.go

```diff
@@ -0,0 +1,194 @@
+package main
```

Comment:
You're being very liberal with usage of exported names. Even if it's just a file in the main package, you're making things harder if you evolve this into a small package later. I'd suggest to still think of it as a micro-package and use lowercase names for everything but main entry points.